app/controllers/baby_cost_calculator_controller.rb

```diff
@@ -1,4 +1,10 @@
 class BabyCostCalculatorController < EmbeddedToolsController
+  before_action :deny_non_syndicates
+
+  def deny_non_syndicates
+    redirect_to main_app.article_path('baby-costs-calculator'), status: 301 if syndicated_tool_request? == false
```

Comment:
I would use `unless syndicated_tool_request?` and then you dont need the check `== false`